daemon/daemon/daemon.go

```diff
@@ -61,27 +64,24 @@ var (
 // Daemon is the cilium daemon that is in charge of perform all necessary plumbing,
 // monitoring when a LXC starts.
 type Daemon struct {
-	ipamConf                  *ipam.IPAMConfig
-	kvClient                  kvstore.KVClient
-	containers                map[string]*types.Container
-	containersMU              sync.RWMutex
-	endpoints                 map[uint16]*endpoint.Endpoint
-	endpointsDocker           map[string]*endpoint.Endpoint
-	endpointsDockerEP         map[string]*endpoint.Endpoint
-	endpointsMU               sync.RWMutex
-	endpointsLearning         map[uint16]labels.LearningLabel
-	endpointsLearningMU       sync.RWMutex
-	endpointsLearningRegister chan labels.LearningLabel
-	events                    chan events.Event
-	dockerClient              *dClient.Client
-	loadBalancer              *types.LoadBalancer
-	k8sClient                 *k8s.Clientset
-	conf                      *Config
-	policy                    policy.Tree
-	consumableCache           *policy.ConsumableCache
-	ignoredContainers         map[string]int
-	ignoredMutex              sync.RWMutex
-	loopbackIPv4              net.IP
+	ipamConf          *ipam.IPAMConfig
+	kvClient          kvstore.KVClient
+	containers        map[string]*types.Container
+	containersMU      sync.RWMutex
+	endpoints         map[uint16]*endpoint.Endpoint
```

Comment:
1) This uint16 will become a string.

2) Maybe add a TODO for this: sense we have different prefixes for each ID (`docker:` vs `cilium:` etc) what about removing these 3 maps all together and having multiple keys pointing for the same value?